x-pack/plugins/spaces/public/nav_control/solution_view_tour/solution_view_tour.tsx

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiButtonEmpty, EuiLink, EuiText, EuiTourStep } from '@elastic/eui';
+import React from 'react';
+import type { FC, PropsWithChildren } from 'react';
+
+import type { OnBoardingDefaultSolution } from '@kbn/cloud-plugin/common';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+import type { SolutionView } from '../../../common';
+
+const tourLearnMoreLink = 'https://ela.st/left-nav';
```

Comment:
>[!IMPORTANT]
> How long do you think we’ll need to rely on this link? Who currently has permissions to edit or delete it? Is your plan to eventually move this to the proper Kibana docs and use the docLinks service instead?
> 
> I’m a bit concerned that we don’t have any guardrails to prevent this link from breaking or pointing to anything that it shouldn't point to. Unlike normal doc links changes to to this short URL doesn't require any review.

Reply:
Thanks, we should be good to go then (wasn't sure if it was going to stay forever).

Reply:
@kevinsweet please correct my assumptions here. If they are correct maybe we could already open the issue to remove the tour in release X?